api-service/api/main.go

```diff
@@ -35,6 +35,7 @@ func getAllMenuItems(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		fmt.Fprintf(w, "Failed to read menu items")
 		w.WriteHeader(http.StatusInternalServerError)
+		return
 	}
 
```

Comment:
should this `w.WriteHeader(http.StatusOK)`?

Reply:
I think the 500 status code seems reasonable, if we can't get the menu items (for some reason) we throw an error and stop

Reply:
Yes, agree on the 500. But I mean on the non-err response right above this
```
json.NewEncoder(w).Encode(res.MenuItemsList)
```

Reply:
ah, gotcha, no need, on the first write to `http.ResponseWriter` it'll set the header to "OK"

https://pkg.go.dev/net/http#ResponseWriter
> // If WriteHeader has not yet been called, Write calls
// WriteHeader(http.StatusOK) before writing the data.